src/core/qtlibtorrent/qbtsession.cpp

```diff
@@ -1276,72 +1276,78 @@ void QBtSession::loadTorrentTempData(QTorrentHandle &h, QString savePath, bool m
         TorrentPersistentData::instance()->saveTorrentPersistentData(h, fsutils::fromNativePath(savePath), magnet);
 }
 
-void QBtSession::mergeTorrents(QTorrentHandle& h_ex, const QString& magnet_uri)
+void QBtSession::mergeTorrents(const QTorrentHandle &h, const QString &magnet_uri)
 {
-    QList<QUrl> new_trackers = misc::magnetUriToTrackers(magnet_uri);
-    bool trackers_added = false;
-    foreach (const QUrl& new_tracker, new_trackers) {
+    QStringList trackers = misc::magnetUriToTrackers(magnet_uri);
+    QStringList urlSeeds = misc::magnetUriToUrlSeeds(magnet_uri);
```

Comment:
Are these two misc:: functions above using somewhere else?
We here twice parse magnet link...